### ISSUE TITLE
Propogate view from toggle button on ChromePicker

### ISSUE THREAD

**Issue by J-Son89** [casesandberg/react-color#584]
The current view state of the ChromeFields input in ChromePicker is not externally accessible and makes it difficult to tie in with other components.

I suggest adding an onToggleView callback to pass the new "view" state as a parameter back up to its container.
e.g:
`if(this.props.onToggleView){
  this.props.onToggleView('hex');
}`

<img width="1628" alt="propsfortogglebtn" src="https://user-images.githubusercontent.com/22799766/52292238-2ece7580-296c-11e9-803a-317085269058.png">

**Comment by casesandberg** [comment]
I agree, that would make a good addition. We would probably want a prop like `defaultColorSpace` for you to set the visible view and then a `onColorSpaceChange` as a callback. Would you be willing to put up a PR for this?

**Comment by J-Son89** [comment]
Hey Case, yep I would be happy to make a PR.
